pages/vi/vi-first-steps.md

```diff
@@ -7,8 +7,7 @@ Welcome to the first steps for becoming an OLE Virtual Intern! We treat these fi
 If you are selected for the internship after completing the steps, you will be officially invited to join the OLE interns team! We’ll add you to the Virtual Interns Gitter chatroom and assign you to a specific team to work on developing and improving OLE’s software. Our current projects could be found [here](#!./pages/robots/rbts-intern-orientation.md#Familiarize_Yourself_with_Current_Projects_and_Issues).
 
 
-Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will have the opportunity to work with software and languages including **[Git](https://git-scm.com/)**, **[GitHub](https://github.com/)**, **[Markdown](https://daringfireball.net/projects/markdown/syntax)**, **[Vagrant](https://www.vagrantup.com/)**, **[VirtualBox](https://www.virtualbox.org/)**, **[Command Line/Terminal](https://en.wikipedia.org/wiki/Command-line_interface)**, **[Command Line/Terminal Scripts](https://www.codecademy.com/articles/command-line-commands)**, **[Vim](https://www.vim.org/)**, **[CouchDB](http://couchdb.apache.org/)**, **[Docker](https://www.docker.com/)**, **[HTML5](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5)**, **[JavaScript](https://en.wikipedia.org/wiki/JavaScript)**, **[Node.js](https://nodejs.org/en/)**, and **[Angular](https://angular.io/)**. To learn more about "How To Code", go to **[Tutorials](https://open-learning-exchange.github.io/#!./pages/vi/vi-faq.md#Helpful_Links)**.
-
```

Comment:
you removed line ahead of **NOTE** as well. So all your content are showing up as note. Please fix this.